src/pages/protectedroute.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useSelector } from "react-redux";
+import Login from './User/Login';
+import Logout from './User/Logout';
+
+const ProtectedRoute = () => {
+  const { isLoggedIn } = useSelector(state => state.auth);
+
+  if (isLoggedIn) {
+    return <Logout />;
+  } else {
+    return <Login />;
+  }
```

Comment:
```suggestion
  return isLoggedIn ? <Logout/> : <Login/>;
```
nit: alternatively, we can use a single ternary for the return unless we plan to have more logic in this component later on

Reply:
Yeah the logic inside this component is tentative - the login/logout components are mostly there for testing purposes right now. I anticipate that we'll be adding more logic around whether the user is logged in. @jzxhuang can you confirm?

Reply:
Yep, I think this is pretty temporary code (placeholder stuff). Agree w/ using ternary statements in React render part but we can make this change later